imagedephi/rules.py

```diff
@@ -9,10 +9,20 @@ class FileFormat(Enum):
     SVS = "svs"
 
 
+expected_type_map: dict[str, list[Type[Any]]] = {
+    "integer": [int],
+    "number": [int, float],
+    "text": [str],
+    "rational": [int],
+}
```

Comment:
I note that with Aperio tags, if we specify that the expected type is text, then it will fail if it could be coerced to a numeric type.  This probably means that if we think an Aperio tag *could* be a string or numeric value, we don't want to specify a check_type for it.  We just need to add this to the documentation when we get to it.